python/jacobiandebug/analyzejacobian.py

```diff
@@ -16,9 +16,9 @@
 
 
 # regular expressions to parse the PETSc debug output
-MfdRE = re.compile("^Finite difference Jacobian \(user-defined state\)")
+MfdRE = re.compile("^Finite[ -]difference Jacobian \(user-defined state\)")
```

Comment:
This change is not necessary for 3.6.1 as they inconsistently only changed the other occurrence of _finite difference_. I'm putting it in just in case they catch this oversight in the next release ;-)